test/acceptance/requirements.txt

```diff
@@ -0,0 +1,5 @@
+git+https://visitor:visitor@pdihub.hi.inet/fiware/fiware-iotqaUtils@develop
```

Comment:
I don't like having requirements like this too much, as `@develop` is a moving picture... Maybe is better to point to a particular release or, at least, a commit hash?